### PR TITLE
AJAX connection between html page and xml file

### PR DESCRIPTION
**Pull request by vh78**

### CODE REVIEW

File: second.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+      myFunction(this);
+    }
+  };
+  xhttp.open("GET", "new.xml", true);
+  xhttp.send();
+}
+function myFunction(xml) {
```
Comment (mgenov) [inline second.html]: A meaningful name would be better as it better describes the intention of the function. 

For example: renderLangTable(xml)

File: second.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+<script>
+function loadDoc() {
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+      myFunction(this);
+    }
+  };
+  xhttp.open("GET", "new.xml", true);
```
Comment (mgenov) [inline second.html]: languages.xml instead of new.xml as it will let the reader of the code know that it contains languages

File: second.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Second task</title>
+<meta charset = "utf-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (mgenov) [inline second.html]: jQuery is not used and these dependencies could be removed for now.

File: second.html (hunk 1 above)
Comment (vh78) [reply second.html]: I will remake this immediately.

File: second.html (hunk 2 above)
Comment (vh78) [reply second.html]: I will remake this immediately.

File: second.html (hunk 3 above)
Comment (vh78) [reply second.html]: I will immediately remove this dependency.

File: second.html (hunk 1 above)
Comment (vh78) [reply second.html]: It has been repaired.

File: second.html (hunk 2 above)
Comment (vh78) [reply second.html]: It has been repaired.

File: second.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+<body>
+
+<h1>Most popular programming language!</h1>
+
+<button type="button" class="btn btn-primary" onclick="loadDoc()">Look them</button>
+<br><br>
+<table id="demo"></table>
+
+<script>
+function loadDoc() {
```
Comment (mgenov) [inline second.html]: loadLanguages

Comment (vh78) [reply second.html]: It has been repaired.

**Review by mgenov — APPROVED** [review]
LGTM!

**Pull request merged**